.github/workflows/llm_integration.yml

```diff
@@ -127,7 +127,7 @@ jobs:
         with:
           python-version: '3.10.x'
       - name: Install pip dependencies
-        run: pip3 install requests numpy
+        run: pip3 install requests numpy huggingface_hub
```

Comment:
why we need to install this?

Reply:
I am using `snapshot_download` function to download adapters on the fly.
https://github.com/deepjavalibrary/djl-serving/pull/1110/files#diff-1348e5a6c0a696af2fa5904cc01e2bd27c060d427dcf3d30997f8f53201c74ccR490